lib/modules/dosomething/dosomething_campaign/dosomething_campaign.theme.inc

```diff
@@ -105,6 +105,43 @@ function dosomething_campaign_preprocess_common_vars(&$vars, &$wrapper) {
   if (isset($vars['content']['zendesk_form'])) {
     $vars['zendesk_form'] = $vars['content']['zendesk_form'];
   }
+
+  // Problem shares
+  $vars['show_problem_shares'] = theme_get_setting('show_problem_shares');
+  $campaign = $vars['campaign'];
+
+  if ($vars['show_problem_shares']) {
+    $problem_share_prompt = variable_get('dosomething_campaign_problem_share_prompt');
+    if ($problem_share_prompt) {
+      $vars['problem_share_prompt'] = $problem_share_prompt;
+    }
+
+    $base_url = url(base_path(), array('absolute'=> TRUE));
+    $campaign_path = url(current_path(), array('absolute' => TRUE));
+
+    // Create the image path to the problem fact statement image generated for the node.
+    $problem_share_image = $base_url . 'profiles/dosomething/modules/dosomething/dosomething_campaign/dosomething_campaign_problem_shares/images/' . $vars['nid'] . '.png';
```

Comment:
@angaither My dev env is getting angry and saying `module_get_path()` is not a defined function.